src/ui/common_components/IconButton.tsx

```diff
@@ -34,11 +34,19 @@ interface IconButtonProps {
 
 export default class IconButton extends React.Component<IconButtonProps> {
   public render() {
-    const { className, iconName, styleReact, tag, text, ...nativeProps } = this.props;
+    const {
+      className,
+      iconName,
+      styleReact,
+      tag,
+      text,
+      ...nativeProps
+    } = this.props;
 
     // Has to be PascalCase, else JSX will think it's a tag named 'tagName'.
     const TagName = tag === 'input' ? 'label' : tag || 'button';
     return (
+      // eslint-disable-next-line @typescript-eslint/ban-ts-comment
       // @ts-ignore
```

Comment:
What's happening with these ignore lines?  Can we just have the eslint line itself or do we really need ts-ignore?

Reply:
ts-ignore is for the TS compiler, while the eslint is for ignoring the ts-ignore for eslint to function.